CodeStarterCamp_Week3/week3_step1.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  week3_step1.swift
+//  CodeStarterCamp_Week3
+//
+//  Created by jjudy on 2023/09/11.
+//
+
+import Foundation
+
+enum Coffee: String {
+    case americano = "americano"
+    case latte = "latte"
+    case macchiato = "macchiato"
+    case flatWhite = "flatWhite"
+    case frappuccino = "frappucino"
+}
+
+class Person {
+    
+    enum Gender {
+        case male
+        case female
+    }
+    
+    var name: String
+    var age: Int
+    var gender: Gender
+    var money: Int
+    lazy var nickName: String = String(self.name.dropFirst(1))
+ 
+    // 이니셜라이저
+    init(name: String, age: Int, gender: Gender, money: Int) {
+        self.name = name
+        self.age = age
+        self.gender = gender
+        self.money = money
+    }
+    
+    func walk() {
+        print("\(nickName), 열심히 걷습니다!")
+    }
+    
+    func goWork() {
+        print("\(nickName), 출근합니다...")
+    }
+    
+    func buyCoffee() {
+        print("\(nickName), 도핑을 위한 커피 구입! 🥳")
+    }
+    
+    func sleep() {
+        print("\(nickName), 잠에 듭니다! 좋은 밤~😴")
+    }
+}
+
+class CoffeeShop {
+    var brandName: String
+    var branchName: String
+    var salesRevenue: Int
+    lazy var baristas: Array<Person> = [Person]()
+    lazy var menu: Dictionary<Coffee, Int> = [Coffee: Int]()
+    lazy var pickUpTable: Array<Coffee> = [Coffee]()
+    
+    // 이니셜라이저
+    init(brandName: String, branchName: String) {
+        self.brandName = brandName
+        self.branchName = branchName
+        self.salesRevenue = 0
+    }
```

Comment:
```suggestion
    var baristas: Array<Person>
    var menu: Dictionary<Coffee, Int> 
    var pickUpTable: Array<Coffee>
    
    // 이니셜라이저
    init(brandName: String, branchName: String) {
        self.brandName = brandName
        self.branchName = branchName
        self.salesRevenue = 0
        self.baristas = []
        self.menu = []
        self.pickUpTable = []
    }
```
lazy는 굳이 필요없을 것 같습니다.